app/components/pages/SubmissionWizard/steps/Editors/EditorsContent.js

```diff
@@ -79,6 +86,45 @@ class EditorsContent extends React.Component {
     )
   }
 
+  handleSuggestedReviewersChanged = event => {
+    this.props.handleChange(event)
+
+    const MAXIMUM = limits.suggestedReviewers.max
+
+    const itemIsBlank = item => item.name + item.email === ''
+    // const anyBlankMandatory = (revs) => itemIsBlank(revs[0]) || itemIsBlank(revs[1]) || itemIsBlank(revs[0])
+
+    const reviewers = this.props.values.suggestedReviewers
+
+    // logic only kicks in for the optional reviewers
+    if (reviewers) {
+      // first count the blanks at the end
+      let numBlanks = 0
+      for (let index = reviewers.length - 1; index > 0; index -= 1) {
```

Comment:
Shouldn't this stop at 2?

Reply:
No as we use it to see if there is a blank line there

Reply:
Don't we need 3 suggested reviewers fields present all the time? At least that was my understanding of it. 
Looking at the rest of the code it seems to me that right now we can have a minimum of 2 suggested reviewers fields (since you don't look at the first one no matter what). This seems a little random, I would either keep only 1 suggested reviewer or 3.

Reply:
Yes we do need 3 - with additional optional 3. Did you want to discuss on the merged  #441 , or on slack?